test/cases/measure/data/testdata/service_instance_cpm_minute_data.json

```diff
@@ -0,0 +1,212 @@
+[
+  {
+    "tag_families": [
+      {
+        "tags": [
+          {
+            "id": {
+              "value": "1"
+            }
+          },
+          {
+            "str": {
+              "value": "entity_1"
+            }
+          },
+          {
+            "str": {
+              "value": "svc_1"
```

Comment:
Could you add an empty service id here to verify the `NULL` group?

Reply:
Added